virttest/utils_misc.py

```
@@ -4698,3 +4695,79 @@ def _md5(fd):
+        if "snp-policy" not in cvm_guest_info:
+            raise exceptions.TestFail("QMP snp-policy not found in query-sev response.")
+        actual_policy = cvm_guest_info["snp-policy"]
+    else:
+        if "policy" not in cvm_guest_info:
+            raise exceptions.TestFail("QMP policy not found in query-sev response.")
+        actual_policy = cvm_guest_info["policy"]
+    if actual_policy != expected_policy:
+        raise exceptions.TestFail(
+            f"QMP cvm policy mismatch: expected {expected_policy}, "
+            "got {actual_policy}"
+        )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)